From 23690d1afd16a04a85e9cc364155ebd6b18f86a7 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 17 Jul 2026 03:00:02 -0700
Subject: [PATCH] Fix pip_pkg version.py path and non-portable `date --utc`
 flag

---
 pip_pkg.sh | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/pip_pkg.sh b/pip_pkg.sh
index e19dbbf..be78e8b 100755
--- a/pip_pkg.sh
+++ b/pip_pkg.sh
@@ -37,10 +37,13 @@ DEST=$(cd "$DEST" && pwd)
 # does not follow them and fails to include TFP on JAX/Numpy.
 pip_tmp="$(mktemp -d)"
 cp -L -R bazel-bin/pip_pkg.runfiles/tensorflow_probability "${pip_tmp}"
-cd "${pip_tmp}/tensorflow_probability"
+cd "${pip_tmp}"
 
+mkdir -p tensorflow_probability/python
 echo >>tensorflow_probability/python/version.py \
-  "if __version__.endswith('dev'): __version__ += '$(date --utc +%Y%m%d)'"
+  "if __version__.endswith('dev'): __version__ += '$(date -u +%Y%m%d)'"
+
+cd ..
 # Pass through remaining arguments (following the first argument, which
 # specifies the output dir) to setup.py, e.g.,
 #  ./pip_pkg /tmp/tensorflow_probability_pkg --gpu --release
-- 
2.39.5 (Apple Git-154)

