From 7b254c108715588e414ad14833702fb2224ea799 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 17 Jul 2026 03:53:28 -0700
Subject: [PATCH] debug

---
 pip_pkg.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pip_pkg.sh b/pip_pkg.sh
index 6a4b4b9..6f974fb 100755
--- a/pip_pkg.sh
+++ b/pip_pkg.sh
@@ -42,6 +42,8 @@ cd "${pip_tmp}"
 mkdir -p tensorflow_probability/python
 echo >>tensorflow_probability/python/version.py \
   "if __version__.endswith('dev'): __version__ += '$(date -u +%Y%m%d)'"
+
+find . -name "setup.py"
 # Pass through remaining arguments (following the first argument, which
 # specifies the output dir) to setup.py, e.g.,
 #  ./pip_pkg /tmp/tensorflow_probability_pkg --gpu --release
-- 
2.39.5 (Apple Git-154)

