From ca3619ec6e0ea4067bae877c88b4cfe5dfb3f780 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 17 Jul 2026 07:01:47 -0700
Subject: [PATCH] Fix windows runfiles path

---
 pip_pkg.sh | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/pip_pkg.sh b/pip_pkg.sh
index c3b8866..9026ba8 100755
--- a/pip_pkg.sh
+++ b/pip_pkg.sh
@@ -36,7 +36,12 @@ DEST=$(cd "$DEST" && pwd)
 # Use cp -L to resolve symlinks added as part of bazel build. Otherwise wheel
 # does not follow them and fails to include TFP on JAX/Numpy.
 pip_tmp="$(mktemp -d)"
-cp -L -R bazel-bin/pip_pkg.runfiles/tensorflow_probability "${pip_tmp}"
+if [[ -d bazel-bin/pip_pkg.exe.runfiles ]]; then
+  RUNFILES_DIR=bazel-bin/pip_pkg.exe.runfiles
+else
+  RUNFILES_DIR=bazel-bin/pip_pkg.runfiles
+fi
+cp -L -R "${RUNFILES_DIR}/tensorflow_probability" "${pip_tmp}"
 cd "${pip_tmp}/tensorflow_probability"
 
 echo >>tensorflow_probability/python/version.py \
-- 
2.39.5 (Apple Git-154)

